tests/integration/remote/invoke/test_lambda_invoke.py

```diff
@@ -0,0 +1,322 @@
+import json
+import uuid
+import base64
+
+from samcli.lib.utils.boto_utils import get_boto_resource_provider_with_config, get_boto_client_provider_with_config
+from samcli.lib.utils.cloudformation import get_resource_summaries
+
+from parameterized import parameterized
+
+from tests.integration.remote.invoke.remote_invoke_integ_base import RemoteInvokeIntegBase
+from tests.testing_utils import run_command
+
+from pathlib import Path
+import pytest
+
+
+@pytest.mark.xdist_group(name="sam_remote_invoke_single_lambda_resource")
+class TestSingleResourceInvoke(RemoteInvokeIntegBase):
+    template = Path("template-single-lambda.yaml")
+
+    @classmethod
+    def tearDownClass(cls):
+        # Delete the deployed stack
+        cls.cfn_client.delete_stack(StackName=cls.stack_name)
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.stack_name = f"{TestSingleResourceInvoke.__name__}-{uuid.uuid4().hex}"
+        RemoteInvokeIntegBase.remote_invoke_deploy_testing_stack(cls.stack_name, cls.template_path)
+        stack_resource_summaries = get_resource_summaries(
+            get_boto_resource_provider_with_config(),
+            get_boto_client_provider_with_config(),
+            cls.stack_name,
+        )
+        cls.stack_resources = {
+            resource_full_path: stack_resource_summary.physical_resource_id
+            for resource_full_path, stack_resource_summary in stack_resource_summaries.items()
+        }
+        cls.cfn_client = get_boto_client_provider_with_config()("cloudformation")
+        cls.lambda_client = get_boto_client_provider_with_config()("lambda")
+
+    def test_invoke_with_event_and_event_file_provided(self):
+        event_file_path = str(self.events_folder_path.joinpath("default_event.json"))
+        command_list = RemoteInvokeIntegBase.get_command_list(
```

Comment:
nit: wouldn't `self.get_command_list` work here?

Reply:
Yup that should work as well, can change that everywhere.